tests/test_all.py

```diff
@@ -379,6 +380,43 @@ def test_no_compat_packages(container):
         ).is_installed
 
 
+@pytest.mark.parametrize(
+    "container",
+    ALL_CONTAINERS,
+    indirect=True,
+)
+def test_bci_eula_exists(container):
+    """Ensure that the BCI eula exists in the container"""
+
+    assert (
+        "SUSE Linux Enterprise Base Container Image License"
+        in container.connection.check_output(
+            "head -n 1 /usr/share/licenses/product/BCI/license.txt"
+        )
+    )
+
+
+@pytest.mark.skipif(
+    OS_VERSION in RELEASED_SLE_VERSIONS
+    or OS_VERSION in ("basalt", "tumbleweed"),
+    reason="BETA EULA not expected",
+)
+@pytest.mark.parametrize(
+    "container",
+    ALL_CONTAINERS,
+    indirect=True,
+)
+def test_sles_beta_eula_exists(container):
+    """Ensure that the SLES Beta eula exists in the container"""
+
+    assert (
+        "SUSE(R) End User License Agreement for Beta Software"
+        in container.connection.check_output(
+            "head -n 1 /usr/share/licenses/product/base/license.txt"
+        )
+    )
```

Comment:
Just wondering whether we shouldn't additionally check that the beta EULA is **not** present in the released SLE version on top of it being present in the unreleased ones?

Reply:
@dcermak yes, I'm planning to do that, but I'm waiting for a reply from legal whether it is permissible to have two contradicting EULAs in the same image. can we do that as a followup?